py/send_message.py

```diff
@@ -1055,11 +1056,13 @@ def _sign_eth_typed_message(self) -> None:
     "message": {
         "from": {
             "name": "Cow",
-            "wallet": "0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826"
+            "wallet": "0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826",
+            "age": 20,
```

Comment:
This ',' should not be here. It makes Python rise a `json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 36 column 9 (char 1153)` error

Reply:
Strange I thought I had tested it. Anyway, removed the comma, thx.